src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -708,7 +708,7 @@ FixedwingPositionControl::set_control_mode_current(const hrt_abstime &now)
 
 		} else if (_pos_sp_triplet.current.type == position_setpoint_s::SETPOINT_TYPE_LAND) {
 
-			if (!_vehicle_status.in_transition_mode) {
+			if (!(_vehicle_status.is_vtol && _param_nav_force_vtol.get() > 0)) {
```

Comment:
With this, you can enter your mode only if you explicitly want to land and make the back transition. That is the most use case i think, but why not try to make it also when back transitioning mid air?